test/integration/containerized/priv_data/env/settings

```diff
@@ -0,0 +1,7 @@
+---
+idle_timeout: 60
+job_timeout: 360
+pexpect_timeout: 10
+process_isolation: true
+process_isolation_executable: podman
+container: ansible/ansible-runner
```

Comment:
@kdelee this look wrong, it was changed but this isn't the syntax it changed to. Worked for me:

```yaml
---
container_image: ansible-execution-env
process_isolation_executable: docker
process_isolation: true
```

https://github.com/ansible/ansible-builder/pull/43

Reply:
> @kdelee this look wrong, it was changed but this isn't the syntax it changed to. Worked for me:

I'm having trouble parsing this. The example you posted does reflect the latest options, though.